proposals/tools.md

```diff
@@ -0,0 +1,90 @@
+This includes code that tests a runtime's conformance to the OCI Container Runtime spec.
+* Image-Tools: Tools for testing of container images implementing the OCI image-spec.  
+This includes code that validates a file's conformance to the OCI Container Image Format spec.  
+
+### Initial Maintainers
+Initial maintainers of the runtime-tools project:
+* Maintainers of the runtime-spec project
+* Three additional most active contributors (by commit count as of Wed, 8/24/2016) to the repo that will become runtime-tools:
+  - Ma Shimiao (https://github.com/Mashimiao)
+  - W. Trevor King (https://github.com/wking)
+  - Liang Chenye (https://github.com/liangchenye)
+
+(Murnal Patel is one of the most active contributors and he will be included with the first bullet point.)
+
+Initial maintainers of the image-tools project shall be:
+* Maintainers of the image-spec project
+* Three additional most active contributors (by commit count as of Wed, 8/24/2016) to the files that will become image-tools:
+  - Sergiusz Urbaniak (https://github.com/s-urbaniak)
+  - Antonio Murdaca (https://github.com/runcom)
+  - xiekeyang (https://github.com/xiekeyang)
+
+(Vincent Batts and Jonathan Boulle are two of the most active contributors and they will be included with the first bullet point.)
+
+### Code of Conduct
+Both of the proposed projects would incorporate (by reference) the OCI Code of Conduct
+
+### Governance and Releases
+Both of the proposed projects would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+Released version numbers of the runtime-tools and image-tools projects should roughly align with released verions of the associated specs projects.
+
+It is expected that releases of the tools projects will follow releases of the specs projects by anywhere from a few days to a few months.
+
+## Frequenty Asked Questions (FAQ)
+Q: What about the runc project?
+A: This proposal would not impact runc.  
+Runc is a reference implementation of the runtime-spec and it lives in its own repo.
+
+Q: What about schemas?
+A: It would be up to the OCI developer community to determine if schemas fit best with the specs or tools projects.
+
+Q: Why are the repo names plural (tools) instead of singular (tool)?
+A: There may be multiple tools or a single tool with multiple functionalities.
+
+Q: Why not prefix the repo names with "oci-" aligning to the CLI commands for invoking the tools?
+A: The repos will be under the http://www.github.com/OpenContainers/ organization.
```

Comment:
If the [flexible repo-naming proposal][1] goes through, we'd want to drop this FAQ.

[1]: https://github.com/opencontainers/tob/pull/18/files#r76834447